### DISCUSSION TITLE
Where is the configuration stored in Mac?

### DISCUSSION THREAD

**Discussion by creke** [LorenEteval/Furious#125] [Q&A]
I can't started it up again after a crash
Think I need to clear the configuration to fresh start.
Would you please tell me the location of the configuration dir in Mac?
Thank you

**Comment by vint2k** [comment]
Same problem after crash.
Restarting the OS helps me, but it seems that this is not a good solution.

**Comment by LorenEteval** [comment]
Try `~/Library/Preferences/com.furious-gui.Furious.plist`.

**Reply by creke** [reply]
> Try `~/Library/Preferences/com.furious-gui.Furious.plist`.

Thank you, I finnaly found the setting file.
But the app still crashes after deletion.
It seems there may be other reasons.

```
/Applications/Furious-GUI.app/Contents/MacOS/Furious-GUI
qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
# crashed/not started
```

**Reply by LorenEteval** [reply]
> > Try `~/Library/Preferences/com.furious-gui.Furious.plist`.
> 
> Thank you, I finnaly found the setting file. But the app still crashes after deletion. It seems there may be other reasons.
> 
> ```
> /Applications/Furious-GUI.app/Contents/MacOS/Furious-GUI
> qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
> qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
> qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
> qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
> qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
> qt.core.qobject.connect: QObject::connect(QObject, Unknown): invalid nullptr parameter
> # crashed/not started
> ```

There's not enough information so I cannot help with that. How does the application crash at first place?
